benchmark/Streamly/Benchmark/Data/Stream/StreamK.hs

```diff
@@ -166,10 +253,12 @@ composeN n f =
         4 -> transform . f . f . f . f
         _ -> undefined
 
+-- XXX We should rename this to scanl'
 {-# INLINE scan #-}
 scan :: Monad m => Int -> Stream m Int -> m ()
 scan n = composeN n $ S.scanl' (+) 0
 
+-- XXX We should use S.map here instead
```

Comment:
Then use a separate benchmark for `fmap`?

Reply:
There's already a separate benchmark for `fmap`